patches/chromium/fix_media_key_usage_with_globalshortcuts.patch

```diff
@@ -59,10 +59,24 @@ index 554930bc33d87ee88a9bcc5f0cf17cef09c27ef0..8df4f91d3db453afb9f73bcaeb82c919
    // true if register successfully, or false if 1) the specificied |accelerator|
    // has been registered by another caller or other native applications, or
 diff --git a/content/browser/media/media_keys_listener_manager_impl.cc b/content/browser/media/media_keys_listener_manager_impl.cc
-index 5938f75742b793868638e693a9a8c8dc686dfc46..1263d679a5174beb960265989c370dd4a58ae7b4 100644
+index 5938f75742b793868638e693a9a8c8dc686dfc46..1137ea6c6b2b14f912b400e3cc43dc6fd0243407 100644
 --- a/content/browser/media/media_keys_listener_manager_impl.cc
 +++ b/content/browser/media/media_keys_listener_manager_impl.cc
-@@ -231,18 +231,24 @@ void MediaKeysListenerManagerImpl::StartListeningForMediaKeysIfNecessary() {
+@@ -55,7 +55,12 @@ bool MediaKeysListenerManagerImpl::StartWatchingMediaKey(
+                                CanActiveMediaSessionControllerReceiveEvents();
+ 
+   // Tell the underlying MediaKeysListener to listen for the key.
+-  if (should_start_watching && media_keys_listener_ &&
++  if (
++#if defined(OS_MAC)
++      !media_key_handling_enabled_ &&
++#endif  // defined(OS_MAC)
++      should_start_watching &&
++      media_keys_listener_ &&
+       !media_keys_listener_->StartWatchingMediaKey(key_code)) {
```

Comment:
`media_keys_listener_` is implemented by `ui/base/accelerators/media_keys_listener_mac.mm` and it provides API for listening for media keys *globally*. In fact, the code in `media_kes_listener_mac.mm` calls `CGEventTapCreate` whenever new media key is watched and this immediately causes an Accessibility popup to appear since this permission is required for `CGEventTapCreate` to function at all.